tests/jsonParser.py

```diff
@@ -5,7 +5,7 @@
 for root, dirs, files in os.walk('activities', topdown=True):
     for name in files:
         filename = os.path.join(root, name)
-        with open(filename) as fp:
+        with open(filename, ) as fp:
```

Comment:
```suggestion
        with open(filename) as fp:
```

this should never be required.